packages/ui-shared/package.json

```diff
@@ -52,6 +54,7 @@
     "js-cookie": "^2.2.1",
     "moment": "^2.29.1",
     "react-command-palette": "^0.12.0-0",
-    "react-router": "4.3.1"
+    "react-router": "4.3.1",
+    "studio-be": "*"
```

Comment:
Maybe we should think of always naming packages @botpress/<package_name>. So that we are consistent across all repos? Also, this helps differentiate NPM dependencies from packages.